Pomodoro.Core/Models/Frequency/FrequencyModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Pomodoro.Core.Models.Frequency
+{
+    public class FrequencyModel
+    {
+        public string FrequencyTypeValue { get; set; } = "None";
```

Comment:
The frequencyValue enum is located in Pomodoro.DataAccess, the model - is in Pomodoro.Core. DataAccess project has the dependency on Core, but we can't create another vice versa, otherwise the Studio will indicate that there is a circular dependency. That is why I will relocate the enum to the Core project.